### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ayamlearning**
"Added the contributors section"

### CODE REVIEW

**Comment by ayamlearning** [comment]
"Task done double check"

File: README.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+            Michael Stockerl
+</a>
+         </td>
+        <td>
+        <img src="https://avatars0.githubusercontent.com/u/20355307?s=400&v=4" width="150px;"/><br />
+         <a
+                href="https://github.com/lustoykov"
+                rel="noopener morefferer"
+                target="_blank"> 
+            Lyubomir lustoykov
```
Comment (mstockerl) [inline README.md]: There is a typo in the last name. It is Lyubomir Stoykov ;)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,56 @@
...
+</a>
+        </td>
+    </tr>
+</table>
```
Comment (mstockerl) [inline README.md]: IMO Pranjal is missing in the list of contributors. ;)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,56 @@
+# Contributors
```
Comment (mstockerl) [inline README.md]: Not sure, whether this should be on the very top of the README. 

@lustoykov What do you think?

File: README.md (hunk 1 above)
Comment (ayamlearning) [reply README.md]: Noted

File: README.md (hunk 3 above)
Comment (mstockerl) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (mstockerl) [reply README.md]: pranjal2394

**Pull request merged**